src/utils/cookieUtils.ts

```diff
@@ -0,0 +1,43 @@
+const includeDomain = process.env.NODE_ENV !== "test"
+
+export const getTimeLeft = (expTime) => {
+  console.log(expTime)
+  return (new Date(expTime).getTime() - new Date().getTime()) / 1000
+}
+/**
+ * setCookieWithMaxAge(sKey, expiration, value)
+ * set a cookie
+ * @param {string} sKey - The name of the cookie
```

Comment:
Code looks good but can you replace the JSDoc in the comment with typescript annotations?